patientsearch/api.py

```diff
@@ -80,11 +80,15 @@ def validate_auth():
 def current_user_id(token):
     """Safe wrapper to lookup logged in user's identifier string for logging"""
     try:
-        user_id = oidc.user_getfield('email')
+        username = oidc.user_getfield('preferred_username')
     except Exception:
```

Comment:
Is there a specific exception we can catch?

Reply:
I haven't been able to reliably reproduce to catch, thus the catchall :(